<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>flipover_widget</class>
 <widget class="QWidget" name="flipover_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>62</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,2,1,1,1,1,1,2,1,1,2">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QLabel" name="allpg_lb">
     <property name="styleSheet">
      <string notr="true">	font-size:30px;
	height:30px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="onepagenumm_cb">
     <property name="styleSheet">
      <string notr="true">	font-size:30px;
	height:30px;</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QToolButton" name="first_bt">
     <property name="styleSheet">
      <string notr="true">	font-size:30px;
	height:30px;</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="previous_bt">
     <property name="styleSheet">
      <string notr="true">	font-size:30px;
	height:30px;</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="current_bt">
     <property name="styleSheet">
      <string notr="true">	font-size:30px;
	height:30px;</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="next_bt">
     <property name="styleSheet">
      <string notr="true">	font-size:30px;
	height:30px;</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="last_bt">
     <property name="styleSheet">
      <string notr="true">	font-size:30px;
	height:30px;</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="jump_lb">
     <property name="styleSheet">
      <string notr="true">	font-size:30px;
	height:30px;</string>
     </property>
     <property name="text">
      <string>跳转到</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="jump_ledit">
     <property name="styleSheet">
      <string notr="true">	font-size:30px;
	height:30px;</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
